bears/r/FormatRBear.py

```diff
@@ -78,13 +78,18 @@ def create_arguments(filename, file, config_file,
             ```
         """
         options = {'source="' + escape(filename, '"\\') + '"',
-                   'blank=' + _map_to_r_bool(r_keep_blank_lines),
-                   'brace.newline=' + _map_to_r_bool(r_braces_on_next_line),
-                   'comment=' + _map_to_r_bool(r_keep_comments),
-                   'arrow=' + _map_to_r_bool(r_use_arrows),
                    'indent=' + str(indent_size)}
         if r_max_expression_length:
             options.add('width.cutoff=' + str(r_max_expression_length))
+        if r_braces_on_next_line is not None:
+            options.add('brace.newline=' +
+                        _map_to_r_bool(r_braces_on_next_line))
+        if r_keep_blank_lines is not None:
+            options.add('blank=' + _map_to_r_bool(r_keep_blank_lines))
+        if r_keep_comments is not None:
+            options.add('comment=' + _map_to_r_bool(r_keep_comments))
+        if r_use_arrows is not None:
+            options.add('arrow=' + _map_to_r_bool(r_use_arrows))
```

Comment:
Same logic goes here also . user may have used both `=` and `->` . if it doesn't  check `is not None` then it will skip `if` statement.
Suppose user provided `False` value.then it will skip `if` and the values won't effect the code.
Correct me if i'm wrong